lib/util/util.js

```diff
@@ -666,7 +669,8 @@ function startLogger() {
     const winstonExtension = {
         /**
          * helper that prints better stack trace for errors
-         * @param {Exception} ex the error
+         *
+         * @param {Error} ex the error
```

Comment:
what happened here?

Reply:
Exception isnt supported -> should be Error